mkosi/__init__.py

```diff
@@ -3175,9 +3166,6 @@ def install_gentoo(
         gentoo.invoke_emerge(args, root, pkgs=gentoo.pkgs_fs)
 
     if not do_run_build_script and args.bootable:
-        # Please don't move, needs to be called before installing dracut
-        # dracut is part of gentoo_pkgs_boot
```

Comment:
what about this comment? no longer valid?

Reply:
Well I mean gentoo still boots fine, so I assume there's no problem. We run kernel-install manually later which runs dracut so all should be good